DEVELOPER.md

```diff
@@ -45,5 +45,11 @@ export sumo_url=https://events.sumologic.net/receiver/v1/http/XXXXXXXXXX
 rspec spec/
 ```
 
-The project is integrated to Travis CI now.
-Make sure [all test passed](https://travis-ci.com/SumoLogic/logstash-output-sumologic) before creating PR
+## Continuous Integration
+
+The project uses GitHub Actions for:
+
+- Testing pull requests and main branch commits: [.github/workflows/ci.yml](.github/workflows/ci.yml)
+- Publishing new version of gem to RubyGems.org after tagging: [.github/workflows/publish.yml](.github/workflows/publish.yml)
+
+Before publishing a new version, make sure the RubyGems account has MFA disabled for API access. Go to [Settings](https://rubygems.org/settings/edit) and set `MFA Level` to `UI only`. RubyGems credentials are stored in 1Password's `K8s Collection vault`.
```

Comment:
I wouldn't add information about 1Password's vaults here since this is a public repo.

```suggestion
Before publishing a new version, make sure the RubyGems account has MFA disabled for API access.
Go to [Settings](https://rubygems.org/settings/edit) and set `MFA Level` to `UI only`.
```